api/db/drop.js

```diff
@@ -5,6 +5,14 @@ const { connectDb, closeDb } = require('./connection');
 
 const Interest = require('../models/Interest');
 const User = require('../models/User');
+const Event = require('../models/Event');
+const Attendant = require('../models/Attendant');
+const { createPrompt } = require('./utils');
+
+const interestCount = async () => await Interest.countDocuments();
+const userCount = async () => await User.countDocuments();
+const eventCount = async () => await Event.countDocuments();
+const attendantCount = async () => await Attendant.countDocuments();
```

Comment:
```suggestion
const getInterestsCount = async () => await Interest.countDocuments();
const getUsersCount = async () => await User.countDocuments();
const getEventsCount = async () => await Event.countDocuments();
const getAttendantsCount = async () => await Attendant.countDocuments();
```
Following conventions, functions should start with verbs. Please note that I also pluralized the nouns.

⚠️ This will require changes elsewhere!